_data/projects/Learning_Object_Oriented_Python.yml

```
@@ -0,0 +1,10 @@
+name: Learning-Object_Oriented-Python
```

Not sure whether this should be standardized on `_` or `-` so I settled on the repository name:

```suggestion
name: Learning-Object-Oriented-Python
```